plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -456,9 +363,9 @@ def showControlInterface(self):
     ##  Send a command to printer. 
     #   \param cmd string with g-code
     def sendCommand(self, cmd):
-        if self.isPrinting():
+        if self._progress:
```

Comment:
Except that in this case this is not done by the API, but by the implementation of the API (not being able to do certain things when printing is specific to USB printing).